spec/features/chitter_features_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'pg'
+
+feature Chitter do
+
+  scenario 'to see all the messages (peeps)' do
+    Peeps.create(message: 'This is a peep!')
+    Peeps.create(message: 'Summer is not here yet')
+    Peeps.create(message: 'ToYoda is the coolest')
+    
+
+    visit '/peeps'
+
+    expect(page).to have_content "This is a peep!"
+    expect(page).to have_content "Summer is not here yet"
+    expect(page).to have_content "ToYoda is the coolest"
+  end
+
+  scenario 'to post a  new message (peep)' do
+    visit('/peeps/new')
+    fill_in('message', with: 'Hangover Sunday')
+    click_button('Peep')
+
+    expect(page).to have_content "Hangover Sunday"
+    expect(page).to have_content "2021-06-27"
```

Comment:
Unfortunately it looks like this test no longer works, as we've passed the date it specified.

I'd suggest that for the future you pass the date to the database for these tests when you create the test entry - that way your tests would still be straightforward - and unlikely to break in the future.

While we're talking about the date field - you could help your fellow developers by including the ALTER TABLE statement you used to add the date column in the db/migrations folder. The db/migrations folder is a standard place for people to look for schema updates.